lib/glimesh/resolvers/streams_resolver.ex

```diff
@@ -38,13 +38,23 @@ defmodule Glimesh.Resolvers.StreamsResolver do
     {:error, "Access denied"}
   end
 
+  def end_stream(_parent, %{stream_id: stream_id}, %{context: %{is_admin: true}}) do
+    stream = Streams.get_stream!(stream_id)
+
+    Streams.end_stream(stream)
+  end
+
   def end_stream(_parent, %{channel_id: channel_id}, %{context: %{is_admin: true}}) do
     channel = Streams.get_channel!(channel_id)
     {:ok, stream} = Streams.end_stream(channel)
 
     {:ok, stream}
   end
 
+  def end_stream(_parent, _args, %{context: %{is_admin: true}}) do
+    {:error, "Must specify channelId or streamId"}
+  end
+
   def end_stream(_parent, _args, _resolution) do
     {:error, "Access denied"}
   end
```

Comment:
Can we get this for `log_stream_metadata` as well?

Reply:
Added stream_id as the only allowed input.